dotcom-rendering/src/components/NewsletterCard.tsx

```diff
@@ -143,31 +201,49 @@ export const NewsletterCard = ({
 			css={groupItemStyle}
 			aria-label={newsletter.name}
 		>
-			<NewsletterDetail text={newsletter.frequency} />
-			<h3>{newsletter.name}</h3>
-			<p>{newsletter.description}</p>
-			<div css={buttonHolderStyle}>
-				<Button
-					{...getButtonInitialAriaAttribute(newsletter.name)}
-					priority="tertiary"
-					size="xsmall"
-					iconSide="left"
-					icon={
-						<>
-							<span className={ICON_PLUS_CLASS}>
-								<SvgPlus />
-							</span>
-							<span className={ICON_TICK_CLASS}>
-								<SvgCheckmark />
-							</span>
-						</>
-					}
-					cssOverrides={buttonStyle}
-					data-newsletter-id={newsletter.identityName}
-					data-role={BUTTON_ROLE}
-				>
-					Sign up
-				</Button>
+			{!!newsletter.illustrationCard && (
+				<div css={illustrationStyle}>
+					<CardPicture
+						imageSize="carousel"
+						alt=""
+						master={newsletter.illustrationCard}
+					/>
+				</div>
+			)}
+			{!newsletter.illustrationCard && (
+				<IllustrationPlaceholder
+					theme={newsletter.theme}
+					aspectHeight={300}
+					aspectWidth={500}
+				/>
+			)}
```

Comment:
This is just personal preference so feel free to ignore!

```suggestion
			{newsletter.illustrationCard ? (
				<div css={illustrationStyle}>
					<CardPicture
						imageSize="carousel"
						alt=""
						master={newsletter.illustrationCard}
					/>
				</div>
			) : (
				<IllustrationPlaceholder
					theme={newsletter.theme}
					aspectHeight={300}
					aspectWidth={500}
				/>
			)}
```